storage/bootstrap.go

```diff
@@ -110,7 +110,8 @@ func (m *bootstrapManager) targetRanges(at time.Time) xtime.Ranges {
 	midPoint := at.
 		Add(-ropts.BlockSize()).
 		Add(-ropts.BufferPast()).
-		Truncate(ropts.BlockSize())
+		Truncate(ropts.BlockSize()).
+		Add(ropts.BlockSize())
```

Comment:
Ah okay, may be worth adding a test here to catch it if there isn't already one?